core/domain/suggestion_registry.py

```diff
@@ -466,6 +466,35 @@ def accept(self, commit_message):
             self.final_reviewer_id, self.target_id, [self.change],
             commit_message, is_suggestion=True)
 
+    def get_all_html_content_strings(self):
+        """Gets all html content strings used in this suggestion.
+
+        Returns:
+            list(str). The list of html content strings.
+        """
+        html_string_list = []
+        html_string_list.append(self.change.translation_html)
+        html_string_list.append(self.change.content_html)
+        return html_string_list
```

Comment:
You could just replace all this with `return [..., ...]`

Reply:
done